HW1/P2/P2b.py

```diff
@@ -1,3 +1,32 @@
 from P2 import *
 
 # Your code here
+from pyspark import SparkContext
+
+def pixel2cord(p):
+    i,j = p
+    x = j/500.0 - 2
+    y = i/500.0 - 2
+    itr = mandelbrot(x,y)
+    return p,itr
+
+if __name__ == '__main__':
+    sc = SparkContext("local", appName="Spark1")
+    sc.setLogLevel('WARN')
+    
+    sidePart = 10
+    numPart = sidePart*sidePart
+    
+    xisrdd = sc.parallelize(xrange(0,2000)) #no partition    
+    cartrdd = xisrdd.cartesian(xisrdd) #generate coordinate
+    
+    balance = cartrdd.partitionBy(numPart)
```

Comment:
The performance looks much much tighter (1.95-2.2), but I'm curious why this makes it bimodel on the extremes ?